gradle/publish.gradle

```diff
@@ -1,14 +1,22 @@
+ext {
+    GROUP_ID = 'com.novoda'
+    ARTIFACT_ID = 'gradle-static-analysis-plugin'
+    WEBSITE = 'https://github.com/novoda/gradle-static-analysis-plugin'
+}
+
 version = '0.5.1'
+group = "$GROUP_ID.$ARTIFACT_ID"
```

Comment:
I wonder if this `group` variable is used. 🤔

Reply:
I'm not sure either: the [documentation](https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_markers) mentions that only `plugin.id` and `plugin.version` are used to index an entry in the portal, but then the mention of [`java-gradle-plugin`](https://guides.gradle.org/implementing-gradle-plugins/#plugin-development-plugin) makes everything more confused

Reply:
True that, I followed this [documentation](https://plugins.gradle.org/docs/publish-plugin) which also just mentions the `version`. Will remove the `group` for now.